README.md

```diff
@@ -1,68 +1,41 @@
-# Administration for TSC + Community Committee collaborative efforts
-This repository is formal efforts of the Node.js project that are overseen and require input from both the [TSC](https://github.com/nodejs/TSC) and [CommComm](https://github.com/nodejs/community-committee).
+# Administration Documents for Node.js
 
-Repositories that fall under the responsibility of the TSC and CommComm:
+This repository is formal efforts of the Node.js project that are overseen by the [TSC](https://github.com/nodejs/TSC) regarding the less technical aspects of the project.
+
+It is also used to discuss:
 
-- This repo
 - [Getting Started](https://github.com/nodejs/getting-started)
 - The private [Moderation](https://github.com/nodejs/moderation) repo (Collaborators-only access)
 
 ## Travel Fund
 
-Each member of the Node.js organization can ask for funding to travel and spread knowledge about and support the Javascript ecosystem and the Foundation.
+Each member of the Node.js organization can ask for funding to travel and spread knowledge about and support the JavaScript ecosystem and the Foundation.
 
 The management of Travel Fund for collaborators is managed by the [OpenJS
 Foundation Cross-Project-Council](https://github.com/openjs-foundation/cross-project-council/blob/HEAD/project-resources/MEMBER_TRAVEL_FUND.md).
 
 ## Collaborator Summit
 
-Every year, Node.js collaborators meet up in one or more Collaborator
-Summit. If you are interested in joining or helping with the
-organization, the management of Collaborators Summit is managed by the [OpenJS
-Foundation Summit team](https://github.com/openjs-foundation/summit).
+Every year, Node.js collaborators meet up in one or more Collaborator Summit. If you are interested in joining or helping with the organization, the management of Collaborators Summit is managed by the [OpenJS Foundation Summit team](https://github.com/openjs-foundation/summit).
```

Comment:
Not sure I like this change, I'd prefer the markdown to be wrapped at a reasonable line length (e.g. 80 chars). But granted it's a bit inconsistent with the rest of the file, I'd like to see the whole file wrapped at a given line length, although feel free to ignore as it's a personal preference.

Reply:
I'm happy to make a follow up change making everything (or nothing) wrap. I tried doing it here but the diff is almost everything.

Reply:
```suggestion
Every year, Node.js collaborators meet up in one or more Collaborator
Summit. If you are interested in joining or helping with the
organization, the management of Collaborators Summit is managed by the [OpenJS
Foundation Summit team](https://github.com/openjs-foundation/summit).
```